Dockerfile

```diff
@@ -41,10 +41,11 @@ RUN echo " ... adding throttling support with ZMQ and CZMQ" \
          && make && make install \
          && apk del automake autoconf \
          && rm -rf /tmp/zeromq* && rm -rf /tmp/czmq* \
-         && rm -rf /var/cache/apk/*
+         && rm -rf /var/cache/apk/* 
 
 # openresty build
 ENV OPENRESTY_VERSION=1.9.7.3 \
+    NAXSI_VERSION=0.53-2 \
```

Comment:
NAXSI has been recentele taken out due to license incompatibility.See latest commit.